server/api.go

```diff
@@ -172,7 +172,11 @@ func (p *Plugin) submitScore(w http.ResponseWriter, r *http.Request) {
 
 	// Thank the user for their feedback when they first answer the survey
 	if isFirstResponse {
-		_, _ = p.CreateBotDMPost(userID, p.buildFeedbackRequestPost())
+		_, err := p.CreateBotDMPost(userID, p.buildFeedbackRequestPost())
+		if err != nil {
+			p.API.LogError("Failed to response feedback user")
+			w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Let's just just an error here but don't write the header. The method should still continue and we don't want to `return` here.